applications/RANSApplication/python_scripts/apply_wall_function_process.py

```diff
@@ -0,0 +1,63 @@
+import KratosMultiphysics
+import KratosMultiphysics.RANSApplication as KratosRANS
+
+
+def Factory(settings, Model):
+    if (not isinstance(Model, KratosMultiphysics.Model)):
+        raise Exception(
+            "expected input shall be a Model object, encapsulating a json string"
+        )
+    if (not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string"
+        )
+    return ApplyWallFunctionProcess(Model, settings["Parameters"])
+
+
+class ApplyWallFunctionProcess(KratosMultiphysics.Process):
```

Comment:
I think we should unify the slip process in the `FluidDynamicsApplication` with this one so to have a unique `ApplyWallBoundaryCondition` process (OFC not for this PR).